src/build/ReferenceTrimmer.targets

```diff
@@ -2,6 +2,20 @@
 <Project>
   <UsingTask TaskName="ReferenceTrimmerTask" AssemblyFile="$(ReferenceTrimmerTaskAssembly)" />
 
+  <PropertyGroup>
+    <ReferenceTrimmerGetUsedAssemblyReferencesFile>$(MSBuildProjectDirectory)\$(IntermediateOutputPath)_ReferenceTrimmer_GetUsedAssemblyReferences.txt</ReferenceTrimmerGetUsedAssemblyReferencesFile>
```

Comment:
This should probably be added to `FileWrites` at some point for a proper `Clean`